src/api/spec/features/webui/attributes_spec.rb

```diff
@@ -0,0 +1,88 @@
+require "browser_helper"
+
+RSpec.feature "Attributes", :type => :feature, :js => true do
+  let!(:user) { create(:confirmed_user) }
+  let!(:attribute_type) { create(:attrib_type_with_namespace) }
+  let!(:attribute) { create(:attrib, project_id: Project.find_by_name(user.home_project_name).id) }
+
+  def add_attribute_with_values(package = nil)
```

Comment:
I gave it a try, but failed. In the end I got errors when trying to use the user variable in the shared_example block.
But that might be solvable. Just didn't want to mess with to many things at once. So if I figure out a reasonable way to do this, I'll updated the test. But not now;-)

And the other reason why I did not use shared_examples is, that so far it seems to me that those are more aimed for inheritance and mixins.